rpcserver.go

```diff
@@ -746,11 +746,25 @@ func (s *rpcServer) DepositAccount(ctx context.Context,
 			"minimum is %d sat/kw", feeRate, chainfee.FeePerKwFloor)
 	}
 
+	bestHeight := atomic.LoadUint32(&s.bestHeight)
+
+	// If provided, determine new expiration value.
+	var expiryHeight uint32
+	switch {
+	case req.GetAbsoluteExpiry() != 0 && req.GetRelativeExpiry() != 0:
```

Comment:
nit: is this case even possible? (given it's a `oneof` in the proto).

Reply:
Ah, you're right, it can't really. Forgot it was a `oneof` when I asked for this case to be added. But doesn't hurt either, so we can just leave it IMO.